crates/nu-engine/src/documentation.rs

```diff
@@ -14,13 +14,6 @@ pub fn get_full_help(
     engine_state: &EngineState,
     stack: &mut Stack,
 ) -> String {
-    let config = stack.get_config(engine_state);
-    let doc_config = DocumentationConfig {
-        no_subcommands: false,
-        no_color: !config.use_ansi_coloring,
-        brief: false,
-    };
-
     let stack = &mut stack.start_capture();
```

Comment:
Not sure about the significance of this line here @IanManske

Reply:
That line is just in case / preventing future bugs.

The `get_documentation` function executes some code, and those commands should not be using the `OutDest`s of the current `stack`. Rather, we want to capture the output of those commands which is why there is a `start_capture`. 

`get_documentation` I think only currently calls `table` and `nu-highlight` which are both not affected by the `stack`'s `OutDest`s. So, there is probably no noticable difference right now.

Reply:
Added a comment in [`dd15b30` (#13531)](https://github.com/nushell/nushell/pull/13531/commits/dd15b3001f4fb9c94521c8c99de27d84f7670f28)